script.js

```diff
@@ -0,0 +1,34 @@
+function addNum()
+{
+     
+ var num1=parseInt(document.addCalc.txt1.value)
+ var num2=parseInt(document.addCalc.txt2.value)
+ var num3=num1+num2
+ document.addCalc.txt3.value=num3
+}
+    
+function subNum()
+{
+     
+ var num1=parseInt(document.addCalc.txt1.value)
+ var num2=parseInt(document.addCalc.txt2.value)
+ var num3=num1-num2
+ document.addCalc.txt3.value=num3
+} 
+
+function multNum()
+{
+     
+ var num1=parseInt(document.addCalc.txt1.value)
+ var num2=parseInt(document.addCalc.txt2.value)
+ var num3=num1*num2
+ document.addCalc.txt3.value=num3
+}   
+function divNum()
+{
+     
+ var num1=parseInt(document.addCalc.txt1.value)
+ var num2=parseInt(document.addCalc.txt2.value)
+ var num3=num1/num2
+ document.addCalc.txt3.value=num3
+}  
```

Comment:
You've got a lot of repeated code since you are using different functions here. You could do this with a single function.

```HTML
<input type="button" value="+" class="buttons" onClick="calculate('+')">
<input type="button" value="-" class="buttons" onClick="calculate('-')">
<input type="button" value="*" class="buttons" onClick="calculate('*')">
<input type="button" value="/" class="buttons" onClick="calculate('/')">
```

```JavaScript
function calculate(operator) {
  var num1 = parseInt(document.addCalc.txt1.value)
  var num2 = parseInt(document.addCalc.txt2.value)
  var result

  switch(operator) {
    case '+':
      result = num1 + num2
      break
    case '-':
      result = num1 - num2
      break
    case '*':
      result = num1 * num2
      break
    case '/':
      result = num1 / num2
      break
    default:
      result = 'Invalid operation'
  }

  document.addCalc.txt3.value = num3
}
```